README.md

```diff
@@ -1,83 +1,87 @@
-# Thunderbird Sieve Extension
+# [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) for [Thunderbird](https://www.thunderbird.net/)

```

Comment:
In order to align with the naming, this should be Thunderbird sieve add-on as the github project name. 

The different naming is mostly due to mozilla's inconsistency. At the beginning there were extension. Then they decided to rename it to addon which was several years later changed to add-on.

From my point of view headings should not contain links. This just looks very strange.

Reply:
Will change as you requested. Should "Sieve" begin with a capital letter? As a side note, your project name seems to be missing the hyphen.